standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -875,7 +875,7 @@ public enum ConfVars {
     EVENT_DB_LISTENER_CLEAN_STARTUP_WAIT_INTERVAL("metastore.event.db.listener.clean.startup.wait.interval",
         "hive.metastore.event.db.listener.clean.startup.wait.interval", 1, TimeUnit.DAYS,
         "Wait interval post start of metastore after which the cleaner thread starts to work"),
-    EVENT_DB_NOTIFICATION_API_AUTH("metastore.metastore.event.db.notification.api.auth",
```

Comment:
not sure if it's good to change the property name that existing in old releases

Reply:
noone used such a name, but a hive alias